feature_tests/src/attrs.rs

```diff
@@ -61,7 +61,7 @@ pub mod ffi {
     }
 
     #[diplomat::opaque]
-    #[diplomat::attr(not(supports = comparators), disable)]
+    #[diplomat::attr(any(c, kotlin, cpp, js), disable)]
```

Comment:
doing this for now to keep generation stable

Reply:
works for me!